fsharp-backend/tests/httpbytestestfiles/injected-icon-post.test

```diff
@@ -0,0 +1,20 @@
+[http-handler POST /]
+request.body |> Bytes.length_v0 |> toString |> String.toBytes_v0
+
+[request]
+POST / HTTP/1.1
+Host: HOST
+Date: Sun, 08 Nov 2020 15:38:01 GMT
+Content-Length: 685
+
+[INJECT_DATA=favicon-32x32.png]
```

Comment:
maybe `IMPORT_DATA_FROM_FILE`?

I'm a little confused though - is this a section (implied by the square brackets), or a variable being replaced in the body.

Reply:
It's a variable, but since I was processing it in the parsing of the test, I made it look the same as elsewhere.
I think I'll make it more like `<IMPORT_DATA_FROM_FILE=xyz.abc>`